webapp/src/reducers.ts

```diff
@@ -215,11 +218,69 @@ interface usersStatusesAction {
         channelID: string,
         userID: string,
         raised_hand?: number,
+        reaction?: Reaction,
         states: { [userID: string]: UserState },
     },
 }
 
-const voiceUsersStatuses = (state: usersStatusesState = {}, action: usersStatusesAction) => {
+interface userReactionsState {
+    [channelID: string]: {
+        reactions: Reaction[],
+    }
+}
+
+const queueReactions = (state: Reaction[], reaction: Reaction) => {
+    const result = state?.length ? [...state] : [];
+    result.push(reaction);
+    if (result.length > 50) {
```

Comment:
I'd suggest making the hardcoded `50` a more visible constant.

Reply:
Yes, good idea, done! (along with the other magic number we might want to adjust)